uber/badge_funcs.py

```diff
@@ -2,27 +2,31 @@
 
 
```

Comment:
The functions here still get called with ``c.NUMBERED_BADGES`` is False, they just never result in an error.